Assets/Scripts/GameService.cs

```diff
@@ -51,7 +51,7 @@ private void Start()
         soundService = new SoundService(soundScriptableObject, audioEffectSource, backgroundMusicSource);
         playerService = new PlayerService(playerPrefab, playerScriptableObject, playerBulletPrefab, playerBulletScriptableObject);
         powerUpService = new PowerUpService(powerUpScriptableObject);
-        enemyService = new EnemyService(enemyPrefab, enemyScriptableObject);
+        enemyService = new EnemyService(enemyScriptableObject);
         vfxService = new VFXService(vfxPrefab);
```

Comment:
each enemy prefab might be different so its not a good idea for enemyPrefab to be the same as well